streamalert_cli/terraform/helpers.py

```diff
@@ -24,3 +29,33 @@ def terraform_check():
                        'your $PATH:\n'
                        '\t$ export PATH=$PATH:/usr/local/terraform/bin')
     return run_command(['terraform', 'version'], error_message=prereqs_message, quiet=True)
+
+def terraform_state_lock(action, region, table):
+    ddb_client = boto3.client('dynamodb', region_name=region)
```

Comment:
boto3 clients should utilize [this](https://github.com/airbnb/streamalert/blob/2746509ccf4513e882623fcd7ea83f1b5b29e6d7/streamalert/shared/helpers/boto.py#L28) helper whenever possible for the config.

[See here for an example](https://github.com/airbnb/streamalert/blob/2746509ccf4513e882623fcd7ea83f1b5b29e6d7/streamalert/shared/helpers/aws_api_client.py#L47).